bundles/extension.rst

```diff
@@ -146,11 +146,12 @@ I/O operations and increases the application performance.
 Your bundles can also add their own classes into this file thanks to the
 ``addClassesToCompile()`` and ``addAnnotatedClassesToCompile()`` methods (both
 work in the same way, but the second one is for classes that contain PHP
-annotations). Define the classes to compile as an array of their fully qualified
-class names::
+annotations). These methods are provied by the ``Extension`` class from the
+HttpKernel component. Define the classes to compile as an array of their
+fully qualified class names::
 
     use AppBundle\Manager\UserManager;
-    use AppBundle\Utils\Slugger;
```

Comment:
Why is this import no longer needed? The example uses `Slugger::class` in addition to `UserManager::class`

Reply:
looks like I deleted that line mistakenly